src/pages/fundamentals/eutxo.mdx

```diff
@@ -293,9 +292,9 @@ also constrained because it doesn't allow for expressing any elaborate logic.
 
 This is where the second form gets more interesting. Cardano allows locking
 funds using a script representing the validation logic that must be satisfied
-to spend funds guarded by the address. We typically call such addresses:
-_script addresses_. Similarly to the first form, the entire script must be
-provided as a witness by any transaction spending from a script address, as
+to spend funds guarded by the address. We typically call such addresses
+"_script addresses_". Similarly to the first form, the entire script must be
+provided as a witness to any transaction spending from a script address, as
```

Comment:
provided as a witness *in* any transaction?

Reply:
previous wording (by) is better imo.

Reply:
Fair enough, perhaps I misunderstood what is being described i.e. if what is meant here is that the the transaction *includes* - as part of its composition - the credentials, "by" is correct )